### ISSUE TITLE
Remote monitor is shift-buildable

### ISSUE THREAD

**Issue by narc0tiq** [narc0tiq/YARM#56]
Thanks to @Thel89 at <https://github.com/narc0tiq/YARM/pull/53#issuecomment-262475231>, we now know that it's possible to shift-click to ghost-place a remote monitor, and robots will successfully build it. 

Harvesting it thereafter does give you the item back, but this is still very weird, so probably we shouldn't allow it.

This issue would be invalidated by #55, as the monitor item would stop being something one carries around in their inventory.

**Comment by Nexela** [comment]
2 Ways to handle this in iterim: (1 is the prefered way! because why add an entity)

1.Keep the item, but remove its place_as_result and use on_put_item event.
```
on_put_item
player=game.players[event.player_index]
if player.cursor_stack.name='yarm-remote' then
--code here
end
```

2. in the on_built_entity function
```
if event.created.entity = entity-ghost and event.created.entity.ghost_name = "Yarmremote" then
entity.destroy()
end
```

See #55 options for that :)

**Comment by narc0tiq** [comment]
Fixed by v0.8 -- no more remote monitor entity means no more ghost-building.
